<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">alternate-background-color: rgb(255, 217, 206);
background-color: rgb(255, 213, 171);</string>
   </property>
   <widget class="QLineEdit" name="LastName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchStudButt">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SeachCourseButt">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Course</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FirstName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="MiddleName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="IDNumber">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Gender">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="YearLevel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>561</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowStudButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Students</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowCourButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Courses</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteStudButt">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>240</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DelCoursButt">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>310</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Course</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CourseCode">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>390</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Course Code</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CourseName">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>440</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Course Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddCoursButt">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>490</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Course</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>761</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>22</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Simple Student Information System&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Middle Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Year Level</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Course Code</string>
    </property>
   </widget>
   <widget class="QComboBox" name="IDNumberCombobox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddStudentButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Student</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>[+] Status</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Search">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>761</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>180</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
